yew-functional/src/use_context_hook.rs

```diff
@@ -115,6 +120,34 @@ where
         .and_then(|scope| scope.get_component().map(|comp| f(&*comp)))
 }
 
+/// Hook for consuming context values in function components.
+/// The context of the type passed as `T` is returned. If there is no such context in scope, `None` is returned.
+/// A component which calls `use_context` will re-render when the data of the context changes.
+///
+/// More information about contexts and how to define and consume them can be found on [Yew's website](https://yew.rs).
+///
+/// # Example
+/// ```rust
+/// # use yew_functional::{function_component, use_context};
+/// # use yew::prelude::*;
+/// # use std::rc::Rc;
+///
+/// # #[derive(Clone, Debug, PartialEq)]
+/// # struct ThemeContext {
+/// #    foreground: String,
+/// #    background: String,
+/// # }
+/// #[function_component(ThemedButton)]
+/// pub fn themed_button() -> Html {
+///     let theme = use_context::<Rc<ThemeContext>>().expect("no ctx found");
+///
+///     html! {
+///         <button style=format!("background: {}; color: {}", theme.background, theme.foreground)>
+///             {"Click me"}
```

Comment:
You forgot to apply the changes from the documentation here too. Note the missing semicolon in `style` as well.
```suggestion
///             { "Click me" }
```